src/ert/config/parsing/config_schema_item.py

```diff
@@ -38,7 +38,7 @@ class SchemaItem:
     # A list of item's which must also be set (if this item is set). (can be NULL)
     required_children: List[str] = Field(default_factory=list)
     # Information about the deprecation if deprecated
-    deprecation_info: Optional[DeprecationInfo] = None
+    deprecations_info: List[DeprecationInfo] = Field(default_factory=list)
```

Comment:
Why plural?

Reply:
Because a keyword can have multiple deprecations. In this context, QUEUE_OPTION is the keyword, and the deprecations can be both LSF_LOGIN_SHELL and LSF_RSH_CMD.

Reply:
Yes, it is just nitpicking, but still think `deprecation_info` was just fine. But we leave it up to you.